src/reanimated2/hook/useHandler.ts

```diff
@@ -66,7 +65,7 @@ export function useHandler<
   const initRef = useRef<ContextWithDependencies<Context> | null>(null);
   if (initRef.current === null) {
     initRef.current = {
-      context: makeRemote<Context>({} as Context),
+      context: {} as Context,
```

Comment:
Could you please explain why is it possible to remove `makeRemote` here? How exactly can we make sure that this change will not break anything?

Reply:
I will add a detailed description in the PR why `makeRemote` is no longer necessary and why are those changes safe.

Reply:
@tomekzaw added!

Reply:
Fixed in #5586.